indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -83,8 +83,10 @@
     };
 
     function refreshPersonFilters() {
+        $('.js-event-person-list #person-filters ul>li').removeClass('enabled');
         var personRows = $('.js-event-person-list tr[data-person-roles]');
         var filters = $('.js-event-person-list [data-filter]:checked').map(function() {
+            $(this).parent('li').addClass('enabled');
```

Comment:
- `var $this = $(this);`
- I'd use `.closest()` instead of `.parent()` to be more resilient to minor changes in the DOM